rbac/management/role/view.py

```diff
@@ -639,6 +641,20 @@ def validate_role(self, request):
                         error = {key: [_(message)]}
                         raise serializers.ValidationError(error)
 
+                # validate permission not being added to workspace out of users org for v1 (RHCLOUD-35481)
+                resourceDefinitions = perm.get("resourceDefinitions")
+                for resourceDefinition in resourceDefinitions:
+                    attributeFilter = resourceDefinition.get("attributeFilter")
+                    if is_resource_a_workspace(app, resource_type, attributeFilter):
+                        workspace_id = get_workspace_id_from_resource_definition(attributeFilter)
```

Comment:
If the value is a list, we have to iterate through all the values instead of getting the first one

Reply:
- [ ] check if value is list